crates/ruff_wasm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ruff_wasm"
-version = "0.0.0"
+version = "0.0.275"
```

Comment:
How to we ensure that the version numbers stay in sync? Where do we read the version number? 

Should we add a public `VERSION` constant to the `ruff` crate and use that version instead (we can re-export it from the wasm crate if necessary)?

Reply:
We just bulk replace them -- I'm not worried about it, but also fine to add a public `VERSION` constant :)

Reply:
(Will change.)

Reply:
I guess it depends on what version we want to say. The version of the ruff engine or the version of the WASM bindings.